packages/service-library/src/servicelib/client_session.py

```diff
@@ -5,26 +5,34 @@
 """
 import logging
 
-from aiohttp import ClientSession, web
+from aiohttp import ClientSession, ClientTimeout, web
 
 from .application_keys import APP_CLIENT_SESSION_KEY
+from .utils import get_http_client_request_total_timeout
 
 log = logging.getLogger(__name__)
 
 
 def get_client_session(app: web.Application) -> ClientSession:
-    """ Lazy initialization of ClientSession
+    """Lazy initialization of ClientSession
 
     Ensures unique session
     """
     session = app.get(APP_CLIENT_SESSION_KEY)
     if session is None or session.closed:
-        app[APP_CLIENT_SESSION_KEY] = session = ClientSession()
+        # it is important to have fast connection handshakes
+        # also requests should be as fast as possible
+        # some services are not that fast to  reply
+        # Setting the time of a request using this client session to 5 seconds totals
+        timeout_settings = ClientTimeout(
+            total=get_http_client_request_total_timeout(), connect=1, sock_connect=1
```

Comment:
I wonder if we could get the settings from the ``app``, as in the other cases. Same reasoning as explained before.

Reply:
Because we still have a mix of "old" and "new" settings in the services where this is used, I think it might not be possible right now